toolkit/tools/pkg/imagecustomizerlib/imagecustomizer.go

```diff
@@ -543,6 +540,16 @@ func customizeOSContents(ctx context.Context, ic *ImageCustomizerParameters) err
 	ctx, span := otel.GetTracerProvider().Tracer(OtelTracerName).Start(ctx, "customize_os_contents")
 	defer span.End()
 
+	resolvedConfig, err := loadBaseConfig(ic.config, ic.configPath)
+	if err != nil {
+		return fmt.Errorf("failed to load base config:\n%w", err)
+	}
+	ic.config.Input.Image.Path = resolvedConfig.InputImagePath
```

Comment:
The final input image and output image are resolved in the `createImageCustomizerParameters` function.